### ISSUE TITLE
autovoice after 20 seconds

### ISSUE THREAD

**Issue by doni-irc** [realraum/r3bot-AutoVoice#1]
Hi, is it possible to set delay about autovoice? I mean autovoice after 20 seconds to joined

**Comment by stefan2904** [comment]
yes: https://github.com/realraum/r3bot-AutoVoice/blob/master/config.py#L78
